packer/ubuntu16.json

```diff
@@ -0,0 +1,41 @@
+{
+    "variables": {
+        "my_machine_type": "e2-medium",
+        "network": "projects/clgcporg2-057/global/networks/default",
+        "diskSizeGb": "20",
+        "vm_name": "instance-abc", 
+        "diskTypes": "pd-balanced"
+    },
+    "builders": [
+        {
+            "type": "googlecompute",
+            "project_id": "{{user `project_id`}}",
+            "image_name": "reddit-full-{{timestamp}}",
+            "image_family": "reddit-full",
+            "source_image_family": "{{user `source_image_family`}}",
+            "image_storage_locations": [
+                "us-central1"
+            ],
+            "zone": "us-central1-a",
+            "ssh_username": "appuser",
+            "machine_type": "{{user `my_machine_type`}}",
+            "disk_size": "{{user `diskSizeGb`}}",
+            "disk_type": "{{user `diskTypes`}}",
+            "network": "{{user `network`}}",
+            "tags": [
+                "puma-server",
+                "http-server",
+                "https-server"
+            ],
+            "image_description": "{{user `vm_name`}}"
+        }
+    ],
+    "provisioners": [
+        {
+            "type": "shell",
+            "script": "scripts/deploy_bake.sh",
```

Comment:
Looks like this file should describe how to build reddit-base template rather than reddit-full so use `deploy.sh` here